### ISSUE TITLE
[Feature Request]  Add ability to add fleet and default payout percentages

### ISSUE THREAD

**Issue by spatialthreat** [ppfeufer/aa-srp#185]
### Context

Bring in feature set for srp payouts similar to SEAT plugin

### Describe the feature you have in mind

Add ability to create default payout scheme.  Example would include for payout math to be altered by percentages for hull, fit, cargo and then subtract platinum insurance.

### Additional context

I'll update this here shortly.

Additional note:  I plan on taking a crack at working this feature for my corp/alliance who is transitioning from SEAT to AA.

**Comment by Bluedagger21** [comment]
Voting for this as well. Would help to automatically set SRP payouts based on how the group wishes to define its SRP. Implementing an ISK limit as well would be useful.

**Comment by aevans1987** [comment]
Agree, Don't have to set fixed values, just allow for a settings option where you can say "X% Hull, Y% Modules, Z% Cargo". Let people set those %s as they want.

**Comment by PhilipMcKie** [comment]
Another +1. This is one of the few features that SeAT still has over AA. Would be nice to include that functionality in AA to further ease some of the burden off of SRP approvers.
